tests/maintainer-missing/default.nix

```diff
@@ -0,0 +1,9 @@
+{ pkgs
+}:
+
+{
+  # positive cases
+  no-maintainer = pkgs.callPackage ./no-maintainer.nix { };
+  empty-maintainer = pkgs.callPackage ./empty-maintainer.nix { };
```

Comment:
Actually the field is called `maintainers` so perhaps we should use plural here.

Reply:
Whoops -- I misspelled it everywhere in fact, even in the check!